packages/ui/src/lib/sidebar/Sidebar.stories.svelte

```diff
@@ -0,0 +1,225 @@
+<script>
+	import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
+
+	import Sidebar, { sidebarOpen } from './Sidebar.svelte';
+	import NotSidebar from './NotSidebar.svelte';
+	import SidebarContent from './SidebarContent.svelte';
+	import SidebarHeader from './SidebarHeader.svelte';
+	import SidebarSection from './SidebarSection.svelte';
+
+	sidebarOpen.set(true);
+</script>
+
+<Meta
+	title="Ui/Sidebar"
+	component={Sidebar}
+	parameters={{
+		layout: 'fullscreen'
+	}}
+/>
+
+<Template let:args>
+	<Sidebar {...args} />
+</Template>
+
+<Story name="Default" />
+
+<Story name="A Dynamic Sidebar">
+	<Sidebar>
+		<div class="w-[300px] bg-core-blue-800 p-4 space-y-4 text-white text-sm">
+			<p>
+				<code>{'<Sidebar>'}</code> is a generic component with it's width, height, and other layout attributes
+				calculated based on it's content.
```

Comment:
```suggestion
				<code>{'<Sidebar>'}</code> is a generic component with its width, height, and other layout attributes
				calculated based on its content.
```
Remove apostrophe, as `it's` is not being used as a contraction of "it is".